api/src/main/resources/messages.properties

```diff
@@ -1793,7 +1793,7 @@ Person.birthdate=Birthdate
 Person.birthdateEstimated=Estimated
 Person.dead=Deceased
 Person.dead.checkboxInstructions=Check if this person is deceased
-Person.dead.causeOfDeathNull=Cause of Death is required for people marked as deceased.
```

Comment:
Is there any reason why you removed the above? I thought you were just going to reuse it or add your own new messages.

Reply:
Yes @dkayiwa 

CauseOfDeathNull does not reflect the new business logic accordingly since it is completely fine for cause of death to be null as long as cause of death (non coded) is set. 
That's why I renamed this property to reflect the current implementation better. In addition,I changed the text of the error message to describe the just explained circumstances

 Does this make sense?